scripts/local-cluster.sh

```diff
@@ -442,6 +525,9 @@ main() {
   subcommand="${2:-}"
 
   case "${command}" in
+  setup_node_local_dns)
+    setup_node_local_dns
+    ;;
```

Comment:
Maybe `local-cluster.sh setup node-local-dns` would be a better option?
Then we could add additional ones, like `setup self-signed`.